### DISCUSSION TITLE
Brotli compression level is too low

### DISCUSSION THREAD

**Discussion by juyuwol** [vercel/community#425] [Idea]
In most cases (at least for my project) gzip is even smaller than brotli. If there are no benefits of using brotli, then why would we use brotli?

<img width="478" alt="br-vs-gzip" src="https://user-images.githubusercontent.com/96001622/156904154-92c0a33b-b724-4be5-a3a6-a247d2dcacfa.png">

I saw [the same opinion on Twitter](https://twitter.com/tkadlec/status/1339955423676403712) as well.

Shouldn't the compression level be increased?

**Comment by Pieparker** [comment]
You're always welcome to [exclusively offer gzip compression](https://vercel.com/docs/concepts/edge-network/compression#compression-with-vercel-edge-network), if the Brotli compression rate is not to your liking. That said, I'll defer this to the Vercel feature suggestions category, in order to understand if other developers are looking for higher compression too!
